### PR TITLE
Mod repackaging

### PR DESCRIPTION
**Pull request by bluepichu**
Repackaging mods so that encoders and decoders are in the same worker.  Also reworked some method names to make their functions more obvious.

### CODE REVIEW

File: mods/creamery/code/manifest.json [hunk 1]
```diff
@@ -3,16 +3,9 @@
 	"developer": "creamery",
 	"description": "Decodes code strings.",
 	"tags": ["code"],
-	"types": ["enc","dec"],
```
Comment (zwade) [inline mods/creamery/code/manifest.json]: why don't you include the types?

**Comment by zwade** [comment]
Glancing through, it looks good, but I will have to pull to really take a look

File: mods/creamery/code/manifest.json (hunk 1 above)
Comment (bluepichu) [inline mods/creamery/code/manifest.json]: I feel like I had a good reason.  Let me take a look.

Comment (bluepichu) [inline mods/creamery/code/manifest.json]: Just kidding.  No idea.  I'll add them back in.

**Comment by zwade** [comment]
Looks good to me

**Pull request merged**